docs/hot-module-replacement.md

```diff
@@ -42,6 +42,20 @@ However, it can be a burden to manually change this URL for production deploys.
 
 With this adjustment, Laravel will do the work for you. If you run 'npm run hot' to enable hot reloading, the function will set the necessary `http://localhost:8080` base url. If, instead, you use `npm run dev` or `npm run production`, it'll use your domain as the base.
 
+### Changing the host
+
+If you wish to hot reload over a local network, or if port `8080` is currently in use, you can set the 'Hot Module Replacement' host and port using the following mix option:
+ 
+```js
+mix.options({
+    hmrOptions: {
+        host: '10.1.1.2',
+        port: '3030'
+    }
+});
+```
+
+Important: Currently, support for the `{{ mix('js/bundle.js') }}` blade helper is pending PR.
```

Comment:
I think this is not relevant right now.

Reply:
The PR request?

Reply:
Yes, `mix()` already get merged, right?

Reply:
Ah sorry, I don't get it. I just realize it's your own PR.